### PR TITLE
BPUB-204 poloniex exchange integration

### PR DESCRIPTION
**Pull request by kubacech**

### CODE REVIEW

File: server_extensions_extra/src/main/java/com/generalbytes/batm/server/extensions/extra/bitcoin/exchanges/poloniex/PoloniexExchange.java [hunk 1]
```diff
@@ -0,0 +1,104 @@
...
+
+    @Override
+    public Wallet getWallet(AccountInfo accountInfo, String currency) {
+        return accountInfo.getWallet();
+    }
+
+    @Override
+    protected String translateFiatCurrencySymbolToExchangeSpecificSymbol(String currency) {
+        if (FiatCurrency.USD.getCode().equalsIgnoreCase(currency)) {
+            return "USDT";
```
Comment (pvyhnal-generalbytes) [inline server_extensions_extra/src/main/java/com/generalbytes/batm/server/extensions/extra/bitcoin/exchanges/poloniex/PoloniexExchange.java:100]: Not sure if this is safe assumption, Poloniex has no fiat currencies, right?
We wanted to implement stablecoins on binance, see note in BPUB-933

Comment (pvyhnal-generalbytes) [inline server_extensions_extra/src/main/java/com/generalbytes/batm/server/extensions/extra/bitcoin/exchanges/poloniex/PoloniexExchange.java:100]: Not sure if this is safe assumption, Poloniex has no fiat currencies, right?
We wanted to implement stablecoins on binance, see note in BPUB-933

**Review by pvyhnal-generalbytes — CHANGES_REQUESTED** [review]

File: server_extensions_extra/src/main/java/com/generalbytes/batm/server/extensions/extra/bitcoin/exchanges/poloniex/PoloniexExchange.java (hunk 1 above)
Comment (kubacech) [reply server_extensions_extra/src/main/java/com/generalbytes/batm/server/extensions/extra/bitcoin/exchanges/poloniex/PoloniexExchange.java]: updated, added better support for stablecoins.

**Comment by kubacech** [comment]
added support for binance stablecoins (BATM-1686)

**Review by pvyhnal-generalbytes — APPROVED** [review]

**Pull request merged**